gravitee-am-ui/src/app/domain/settings/certificates/certificates.component.ts

```diff
@@ -139,7 +157,10 @@ export class DomainSettingsCertificatesComponent implements OnInit {
 
   rotateCertificate() {
     this.dialogService
-      .confirm('Rotate key', 'Are you sure you want to generate a new system certificate ?')
+      .confirm(
+        'Rotate key',
+        'Are you sure you want to generate a new system certificate?\nThis will only effect the latest System Certificate created.',
```

Comment:
```suggestion
        'Are you sure you want to generate a new system certificate?\nThis will only affect the latest System Certificate created.',
```
Your change is consistent with figma design, but 'effect' looks strange to me :thinking: